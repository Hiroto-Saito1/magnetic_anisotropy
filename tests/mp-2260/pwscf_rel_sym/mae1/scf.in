! generated by mcif2qewan.py
&control
  calculation = 'scf'
  prefix = 'pwscf'
  pseudo_dir = '/home4/koretsune/PWSCF/pseudo_psl031'
  outdir = './work'
  tstress = .true.
  tprnfor = .true.
  wf_collect = .true.
  disk_io = 'low'
/
&system
  nat = 2
  ntyp = 2
  ecutwfc = 64.0
  ecutrho = 782.0
  occupations = 'smearing'
  smearing = 'm-p'
  degauss = 0.01
  ibrav = 6
  A = 2.728143
  C = 3.778984
  nspin = 2
  starting_magnetization(1) = 3.0
  starting_magnetization(2) = 3.0
/
&electrons
  mixing_beta = 0.1
  conv_thr = 1.d-8
/
ATOMIC_SPECIES
Fe   55.845000  Fe.pbe-spn-rrkjus_psl.0.2.1.UPF
Pt   195.084000  Pt.pbe-n-rrkjus_psl.0.1.UPF

ATOMIC_POSITIONS {crystal}
Fe      0.0000000000     0.0000000000     0.0000000000
Pt      0.5000000000     0.5000000000     0.5000000000

K_POINTS {automatic}
13 13 9 0 0 0
