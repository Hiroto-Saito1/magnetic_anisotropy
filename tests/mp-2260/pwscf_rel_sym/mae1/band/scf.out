
     Program PWSCF v.7.0 starts on 25Aug2023 at 16:43:10 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on    12 processors

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      12
     57152 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         137      45     14                 5843     1095     188
     Max         138      46     16                 5848     1100     189
     Sum        1649     545    177                70151    13173    2261
 
     Using Slab Decomposition
 


     bravais-lattice index     =            6
     lattice parameter (alat)  =       5.1554  a.u.
     unit-cell volume          =     189.8042 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     number of electrons       =        26.00
     number of Kohn-Sham states=           17
     kinetic-energy cutoff     =      64.0000  Ry
     charge density cutoff     =     782.0000  Ry
     scf convergence threshold =      1.0E-08
     mixing beta               =       0.1000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   5.155443  celldm(2)=   0.000000  celldm(3)=   1.385185
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.385185 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  0.721925 )  


     PseudoPot. # 1 for Fe read from file:
     /home/koretsune/PWSCF/pseudo_psl031/Fe.pbe-spn-rrkjus_psl.0.2.1.UPF
     MD5 check sum: ffcf0347a832ff25cbd6031c7d0b600f
     Pseudo is Ultrasoft + core correction, Zval = 16.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.99 svn rev. 10869
     Using radial grid of 1191 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for Pt read from file:
     /home/koretsune/PWSCF/pseudo_psl031/Pt.pbe-n-rrkjus_psl.0.1.UPF
     MD5 check sum: ce1107d01e17cc1780a3d2e71dd4890e
     Pseudo is Ultrasoft + core correction, Zval = 10.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.99 svn rev. 10869
     Using radial grid of 1277 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Fe            16.00    55.84500     Fe( 1.00)
        Pt            10.00   195.08400     Pt( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Fe           1.000
        Pt           1.000

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Fe  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Pt  tau(   2) = (   0.5000000   0.5000000   0.6925927  )

     number of k points=   140  Methfessel-Paxton smearing, width (Ry)=  0.0100

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:    70151 G-vectors     FFT dimensions: (  45,  45,  64)

     Smooth grid:    13173 G-vectors     FFT dimensions: (  27,  27,  40)

     Estimated max dynamical RAM per process >      38.19 MB

     Estimated total dynamical RAM >     458.26 MB

     Check: negative core charge=   -0.000005
     Generating pointlists ...
     new r_m :   0.4083 (alat units)  2.1049 (a.u.) for type    1
     new r_m :   0.4083 (alat units)  2.1049 (a.u.) for type    2

     Initial potential from superposition of free atoms

     starting charge      25.9994, renormalised to      26.0000
     Starting wfcs are   22 randomized atomic wfcs

     total cpu time spent up to now is        1.4 secs

     Self-consistent Calculation

     iteration #  1     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.9

     negative rho (up, down):  0.000E+00 4.498E-05

     total cpu time spent up to now is        3.9 secs

     total energy              =    -323.49889039 Ry
     estimated scf accuracy    <       4.26926901 Ry

     total magnetization       =     6.30 Bohr mag/cell
     absolute magnetization    =     6.31 Bohr mag/cell

     iteration #  2     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  1.1

     negative rho (up, down):  0.000E+00 9.313E-05

     total cpu time spent up to now is        5.9 secs

     total energy              =    -325.55644156 Ry
     estimated scf accuracy    <       3.38503170 Ry

     total magnetization       =     6.01 Bohr mag/cell
     absolute magnetization    =     6.01 Bohr mag/cell

     iteration #  3     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  1.2

     negative rho (up, down):  0.000E+00 1.015E-04

     total cpu time spent up to now is        7.8 secs

     total energy              =    -326.07720847 Ry
     estimated scf accuracy    <       2.11227868 Ry

     total magnetization       =     4.93 Bohr mag/cell
     absolute magnetization    =     5.00 Bohr mag/cell

     iteration #  4     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  8.12E-03,  avg # of iterations =  1.0

     negative rho (up, down):  0.000E+00 1.838E-04

     total cpu time spent up to now is        9.7 secs

     total energy              =    -326.93817768 Ry
     estimated scf accuracy    <       0.97329213 Ry

     total magnetization       =     4.90 Bohr mag/cell
     absolute magnetization    =     4.95 Bohr mag/cell

     iteration #  5     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  3.74E-03,  avg # of iterations =  1.4

     negative rho (up, down):  0.000E+00 1.879E-04

     total cpu time spent up to now is       11.7 secs

     total energy              =    -327.12196469 Ry
     estimated scf accuracy    <       0.45649873 Ry

     total magnetization       =     3.34 Bohr mag/cell
     absolute magnetization    =     3.56 Bohr mag/cell

     iteration #  6     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.76E-03,  avg # of iterations =  1.1

     negative rho (up, down):  0.000E+00 1.607E-04

     total cpu time spent up to now is       13.6 secs

     total energy              =    -327.00485929 Ry
     estimated scf accuracy    <       0.52471417 Ry

     total magnetization       =     3.37 Bohr mag/cell
     absolute magnetization    =     3.56 Bohr mag/cell

     iteration #  7     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.76E-03,  avg # of iterations =  1.9

     negative rho (up, down):  0.000E+00 1.617E-04

     total cpu time spent up to now is       15.7 secs

     total energy              =    -327.07956266 Ry
     estimated scf accuracy    <       0.03265353 Ry

     total magnetization       =     3.43 Bohr mag/cell
     absolute magnetization    =     3.63 Bohr mag/cell

     iteration #  8     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.26E-04,  avg # of iterations =  2.0

     negative rho (up, down):  0.000E+00 1.583E-04

     total cpu time spent up to now is       17.8 secs

     total energy              =    -327.07942649 Ry
     estimated scf accuracy    <       0.02204497 Ry

     total magnetization       =     3.39 Bohr mag/cell
     absolute magnetization    =     3.59 Bohr mag/cell

     iteration #  9     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  8.48E-05,  avg # of iterations =  1.2

     negative rho (up, down):  0.000E+00 1.365E-04

     total cpu time spent up to now is       19.7 secs

     total energy              =    -327.07727491 Ry
     estimated scf accuracy    <       0.01065744 Ry

     total magnetization       =     3.35 Bohr mag/cell
     absolute magnetization    =     3.55 Bohr mag/cell

     iteration # 10     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.10E-05,  avg # of iterations =  2.3

     negative rho (up, down):  0.000E+00 1.306E-04

     total cpu time spent up to now is       21.9 secs

     total energy              =    -327.07715103 Ry
     estimated scf accuracy    <       0.00584985 Ry

     total magnetization       =     3.33 Bohr mag/cell
     absolute magnetization    =     3.53 Bohr mag/cell

     iteration # 11     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  2.25E-05,  avg # of iterations =  2.1

     negative rho (up, down):  0.000E+00 1.267E-04

     total cpu time spent up to now is       24.2 secs

     total energy              =    -327.07929046 Ry
     estimated scf accuracy    <       0.00609467 Ry

     total magnetization       =     3.36 Bohr mag/cell
     absolute magnetization    =     3.55 Bohr mag/cell

     iteration # 12     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  2.25E-05,  avg # of iterations =  1.0

     negative rho (up, down):  0.000E+00 1.237E-04

     total cpu time spent up to now is       26.1 secs

     total energy              =    -327.07858425 Ry
     estimated scf accuracy    <       0.00410452 Ry

     total magnetization       =     3.33 Bohr mag/cell
     absolute magnetization    =     3.52 Bohr mag/cell

     iteration # 13     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.58E-05,  avg # of iterations =  1.6

     negative rho (up, down):  0.000E+00 1.231E-04

     total cpu time spent up to now is       28.1 secs

     total energy              =    -327.07906923 Ry
     estimated scf accuracy    <       0.00010594 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 14     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.07E-07,  avg # of iterations =  3.8

     negative rho (up, down):  0.000E+00 1.211E-04

     total cpu time spent up to now is       30.6 secs

     total energy              =    -327.07905745 Ry
     estimated scf accuracy    <       0.00010598 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.48 Bohr mag/cell

     iteration # 15     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  4.07E-07,  avg # of iterations =  2.0

     negative rho (up, down):  0.000E+00 1.213E-04

     total cpu time spent up to now is       32.7 secs

     total energy              =    -327.07909109 Ry
     estimated scf accuracy    <       0.00011030 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 16     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     ethr =  4.07E-07,  avg # of iterations =  1.1

     negative rho (up, down):  0.000E+00 1.215E-04

     total cpu time spent up to now is       34.6 secs

     total energy              =    -327.07912764 Ry
     estimated scf accuracy    <       0.00000496 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 17     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.91E-08,  avg # of iterations =  2.6

     negative rho (up, down):  0.000E+00 1.219E-04

     total cpu time spent up to now is       37.0 secs

     total energy              =    -327.07914964 Ry
     estimated scf accuracy    <       0.00000151 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 18     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  5.79E-09,  avg # of iterations =  1.8

     negative rho (up, down):  0.000E+00 1.221E-04

     total cpu time spent up to now is       39.0 secs

     total energy              =    -327.07916548 Ry
     estimated scf accuracy    <       0.00000087 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 19     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  3.33E-09,  avg # of iterations =  1.7

     negative rho (up, down):  0.000E+00 1.224E-04

     total cpu time spent up to now is       41.0 secs

     total energy              =    -327.07918000 Ry
     estimated scf accuracy    <       0.00000041 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 20     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  1.60E-09,  avg # of iterations =  2.0

     negative rho (up, down):  0.000E+00 1.226E-04

     total cpu time spent up to now is       43.3 secs

     total energy              =    -327.07919264 Ry
     estimated scf accuracy    <       0.00000022 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 21     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  8.38E-10,  avg # of iterations =  1.6

     negative rho (up, down):  0.000E+00 1.227E-04

     total cpu time spent up to now is       45.3 secs

     total energy              =    -327.07920268 Ry
     estimated scf accuracy    <       0.00000002 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     iteration # 22     ecut=    64.00 Ry     beta= 0.10
     Davidson diagonalization with overlap
     ethr =  6.53E-11,  avg # of iterations =  3.3

     negative rho (up, down):  0.000E+00 1.239E-04
 
     Magnetic moment per site  (integrated on atomic sphere of radius R)
     atom   1 (R=0.408)  charge= 14.7389  magn=  2.9860
     atom   2 (R=0.408)  charge=  7.9192  magn=  0.3852

     total cpu time spent up to now is       47.8 secs

     End of self-consistent calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     the Fermi energy is    16.9627 ev

!    total energy              =    -327.07924498 Ry
     estimated scf accuracy    <          5.4E-09 Ry
     smearing contrib. (-TS)   =      -0.00006681 Ry
     internal energy E=F+TS    =    -327.07917817 Ry

     The total energy is F=E-TS. E is the sum of the following terms:
     one-electron contribution =    -123.83653561 Ry
     hartree contribution      =      83.03390913 Ry
     xc contribution           =     -66.55399151 Ry
     ewald contribution        =    -219.72256019 Ry

     total magnetization       =     3.30 Bohr mag/cell
     absolute magnetization    =     3.49 Bohr mag/cell

     convergence has been achieved in  22 iterations

     negative rho (up, down):  0.000E+00 1.239E-04

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    2 type  2   force =     0.00000000    0.00000000    0.00000000

     Total force =     0.000000     Total SCF correction =     0.000000


     Computing stress (Cartesian axis) and pressure


     negative rho (up, down):  0.000E+00 1.239E-04
          total   stress  (Ry/bohr**3)                   (kbar)     P=        3.91
   0.00004555   0.00000000   0.00000000            6.70        0.00        0.00
   0.00000000   0.00004555   0.00000000            0.00        6.70        0.00
   0.00000000   0.00000000  -0.00001133            0.00        0.00       -1.67


     Writing all to output data dir ./work/pwscf.save/
 
     init_run     :      0.52s CPU      1.22s WALL (       1 calls)
     electrons    :     22.13s CPU     46.44s WALL (       1 calls)
     forces       :      0.16s CPU      0.20s WALL (       1 calls)
     stress       :      0.32s CPU      0.34s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.39s CPU      0.95s WALL (       1 calls)
     potinit      :      0.02s CPU      0.03s WALL (       1 calls)
     hinit0       :      0.06s CPU      0.14s WALL (       1 calls)

     Called by electrons:
     c_bands      :     16.83s CPU     36.56s WALL (      22 calls)
     sum_band     :      4.33s CPU      8.82s WALL (      22 calls)
     v_of_rho     :      0.14s CPU      0.18s WALL (      23 calls)
     newd         :      0.82s CPU      0.85s WALL (      23 calls)
     mix_rho      :      0.02s CPU      0.04s WALL (      22 calls)

     Called by c_bands:
     init_us_2    :      0.19s CPU      0.32s WALL (   13160 calls)
     init_us_2:cp :      0.18s CPU      0.31s WALL (   13160 calls)
     cegterg      :     16.19s CPU     35.65s WALL (    6160 calls)

     Called by *egterg:
     cdiaghg      :      1.68s CPU      2.63s WALL (   17580 calls)
     h_psi        :     13.61s CPU     31.55s WALL (   17860 calls)
     s_psi        :      0.15s CPU      0.30s WALL (   17860 calls)
     g_psi        :      0.01s CPU      0.02s WALL (   11420 calls)

     Called by h_psi:
     h_psi:calbec :      0.57s CPU      1.26s WALL (   17860 calls)
     vloc_psi     :     12.83s CPU     29.84s WALL (   17860 calls)
     add_vuspsi   :      0.18s CPU      0.38s WALL (   17860 calls)

     General routines
     calbec       :      0.84s CPU      1.83s WALL (   25420 calls)
     fft          :      0.08s CPU      0.17s WALL (     613 calls)
     ffts         :      0.00s CPU      0.02s WALL (      90 calls)
     fftw         :     14.95s CPU     35.56s WALL (  577862 calls)
     interpolate  :      0.01s CPU      0.02s WALL (      46 calls)
 
     Parallel routines
 
     PWSCF        :     23.36s CPU     50.25s WALL

 
   This run was terminated on:  16:44: 0  25Aug2023            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
